### PR TITLE
fix(tools): adjust ownership of files

### PR DESCRIPTION
**Pull request by raisedadead**
Adjusts the ownership of files. We will also shortly consider adding the @freeCodeCamp/moderators team to a list of files, but wanted to check in with you all first. 

Note, the goal for this PR is to hopefully result in less noise for the upcoming renovate-bot PRs.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (ojeytonwilliams) [inline .github/CODEOWNERS:20]:
```suggestion
```
I don't think the negation is happening.  If you got to https://github.com/freeCodeCamp/freeCodeCamp/blob/main/LICENSE.md it's still owned by dev-team.

File: .github/CODEOWNERS
Comment (ojeytonwilliams) [inline .github/CODEOWNERS:32]:
```suggestion
```
Same argument as above.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (raisedadead) [reply .github/CODEOWNERS:20]: Thanks for taking the time to review. 

Firstly, this only works when the change (this PR) lands on main. 

Secondly negating things CODEOWNERS file is bit of a blackhole as seen in this old discussion: https://github.community/t/1423, we won't know until this PR lands.

Comment (ojeytonwilliams) [reply .github/CODEOWNERS:20]: Right, but the existing CODEOWNERS file already uses the filename without an owner pattern. If it worked then `package.json`, `LICENSE.md` and so on wouldn't have owners, but they're all owned by dev-team.

Comment (raisedadead) [reply .github/CODEOWNERS:20]: This PR adds the change so that they will be accounted for. Like I said it's a blackhole, if you have suggestions as an update to this PR, I am all ears.

Comment (ojeytonwilliams) [reply .github/CODEOWNERS:20]: Yes and no.  We could just try merging this and seeing if it works.

I'm not 100% clear on the objective, though.  Presumably we want to be able to auto-merge renovate PRs, but if someone creates a PR that modifies the package or package-lock, we'd want a dev-team review before it can go in.

For the md files, it seems fine to have dev-team as owners (or staff, maybe?).

Comment (raisedadead) [reply .github/CODEOWNERS:20]: We have several checks in place including multiple levels of human reviews for each commit that goes into the `prod-current` branch (which is what matters 🤷🏽)

Finally, the current merge requirements (Unfortunately, I do not have the time to explain those) are enough for catching anything out of the ordinary.

IMHO, we have [quite a few things to worry about](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc+-label%3A%22status%3A+on+the+roadmap%22) at the minute.

The objective of this PR is to draw attention to things where it is needed. The above list of issues is one such place to start.

**Comment by ShaunSHamilton** [comment]
Mrugesh, I have looked through Renovate, but feel like I am missing the point; how is it different to camperbot?
Would you mind providing the backstory to why it has been included?

File: .github/CODEOWNERS
Comment (ojeytonwilliams) [reply .github/CODEOWNERS:20]: RIghto. I'm happy to try this approach and see if it does what we want.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by ojeytonwilliams** [comment]
> Mrugesh, I have looked through Renovate, but feel like I am missing the point; how is it different to camperbot?

Dependabot? There's a bit more context for why we stopped using it [here](https://github.com/freeCodeCamp/freeCodeCamp/pull/41487).

**Comment by raisedadead** [comment]
> I have looked through Renovate, but feel like I am missing the point; how is it different to camperbot?

I am assuming you meant [`dependabot`](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically) and not our buddy [camperbot](https://github.com/camperbot).

**Backstory:**

`dependabot` does not meet our requirements anymore https://github.com/freeCodeCamp/freeCodeCamp/pull/41487. If anything, it's a pain.

I am hoping with renovate we are able to meet those requirements including batched updates for mono-repo based projects like Gatsby. The only benefit that `dependabot` previously brought was [automatic merges with a smart approval](https://github.com/ahmadnassri/action-dependabot-auto-merge) from a person with write access.

Now that it requires us to use [a less secure form](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target) of approving the PR, we might as well go with something that covers more ground with saner defaults like renovate.

I do not have a very clear explanation yet because we are still experimenting but it's better than no updates if left to procrastinating humans like yours truly

**Comment by ShaunSHamilton** [comment]
😆 Yes...I meant dependabot.

Thank you, for the explanation. I just missed the removal of dependabot.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Pull request merged**
